modules/andes-core/broker/src/main/java/org/wso2/andes/server/handler/BasicRejectMethodHandler.java

```diff
@@ -127,15 +127,15 @@ public void methodReceived(AMQStateManager stateManager, BasicRejectBody body, i
             }
             else
             {
-                _logger.warn("Dropping message as requeue not required and there is no dead letter queue");
                 try {
                     DeliverableAndesMetadata andesMetadata = AndesUtils.lookupDeliveredMessage(message.getMessage()
                             .getMessageNumber(), channel.getId());
                     Andes.getInstance().moveMessageToDeadLetterChannel(andesMetadata, message
                             .getQueue()
                             .getName());
                 } catch (AndesException e) {
-                    _logger.error("Error while moving message to DLC" , e);
+                    _logger.error("Error while moving message to DLC message Id = "
```

Comment:
why are we using variable names starting with "_"? is it a standard? I feel it will be great if we can follow a single standard.